src/main/java/org/broadinstitute/hellbender/engine/filters/OverclippedReadFilter.java

```diff
@@ -14,9 +14,9 @@
  *     This filter is intended to filter out reads that are potentially from foreign organisms.
  *     From experience with sequencing of human DNA we have found cases of contamination by bacterial
  *     organisms; the symptoms of such contamination are a class of reads with only a small number
- *     of aligned bases and additionally many soft-clipped bases.  This filter is intended
- *     to remove such reads. Consecutive soft-clipped blocks are treated as a single block
- * </p>
+ *     of aligned bases and additionally many soft-clipped bases. This filter is intended
+ *     to remove such reads.
```

Comment:
I changed "Filter out reads that are over-soft-clipped" to "Filter out reads where the number of bases without soft-clips (M, I, X, and = CIGAR operators) is lower than a threshold."